internal/internal_task_pollers.go

```diff
@@ -809,6 +809,8 @@ func (wtp *workflowTaskPoller) poll(ctx context.Context) (interface{}, error) {
 
 	wtp.updateBacklog(request.TaskList.GetKind(), response.GetBacklogCountHint())
 
+	wtp.logger.Info(fmt.Sprintf("Tasklist backlog count is %d", response.GetBacklogCountHint()))
```

Comment:
should we also include which tasklist is this or is it evident from other attributes of the log?

Reply:
That code is excessive, will remove in next commit